services/web/server/src/simcore_service_webserver/director_v2_handlers.py

```diff
@@ -26,42 +31,65 @@
 @permission_required("project.read")
 @log_decorator(logger=log)
 async def start_pipeline(request: web.Request) -> web.Response:
-    director2_settings: Directorv2Settings = get_settings(request.app)
+    client = DirectorV2ApiClient(request.app)
+
+    run_policy = get_run_policy(request.app)
```

Comment:
```
get_project_run_policy
```
minor: And actually, we are only talking about the computations right?

Reply:
yes